tools/change_namespace/AddPaths.cpp

```diff
@@ -0,0 +1,119 @@
+#include "ChangeNamespaceImpl.hpp"
+#include "FileHandler.hpp"
+
+#include <iostream>
+#include <regex>
+
+void ChangeNamespaceImpl::add_path(const fs::path& p)
+{
+   fs::path normalized_path = p;
+   normalized_path = normalized_path.lexically_normal();
+   if(fs::exists(m_cppms_path / normalized_path))
+   {
+      if(fs::is_directory(m_cppms_path / normalized_path))
+         add_directory(normalized_path);
+      else
+         add_file(normalized_path);
+   }
+   else
+   {
+      std::cerr << "CAUTION: dependent file " << p.string() << " does not exist." << std::endl;
+      std::cerr << "   Found while scanning file " << m_dependencies[p].string() << std::endl;
+   }
+}
+
+void ChangeNamespaceImpl::add_directory(const fs::path& p)
+{
+   // Parse files and directories
+   fs::directory_iterator i(m_cppms_path / p);
```

Comment:
Should be use descriptive variable names for better readability and maintainability?